docs/alerts/monitors/index.md

```diff
@@ -37,6 +37,10 @@ Notifications are optional and available as an **alert** and **recovery** for ea
 
 ## Limitations
 
+:::note
+You cannot share monitors, you must duplicate the monitor under another user.
```

Comment:
I'd rephrase as:
You cannot share monitors; instead, you must duplicate the monitor under another user.

Reply:
Thanks Megha! Updating the sentence, now.

Reply:
Hey @Melvin-CnC you can update the changes right on this page https://github.com/SumoLogic/sumologic-documentation/pull/1487/files
![Screenshot 2023-01-27 at 8 10 13 AM](https://user-images.githubusercontent.com/4722362/215134902-50462e67-70bc-4309-9e66-439f0f3337ba.png)

Reply:
Was this change signed off by the PM @Melvin-CnC?